app/javascript/packs/event_tracking.js

```diff
@@ -4,6 +4,7 @@ import { logEvent } from './../packs/log_event';
   'form:update',
   'member:set',
   'member:reset',
+  'petition:submitted',
```

Comment:
unrelated addition - had meant to put this a different branch. It's just proxying another event for gdpr experiments.